doc/sphinx/proof-engine/tactics.rst

```diff
@@ -1577,33 +1577,73 @@ Controlling the proof flow
       While :tacn:`pose proof` expects that no existential variables are generated by
       the tactic, :tacn:`epose proof` removes this constraint.
 
-.. tacv:: specialize (@ident {* @term}) {? as @simple_intropattern}
-          specialize @ident with @bindings {? as @simple_intropattern}
-   :name: specialize; _
-
-   This tactic works on local hypothesis :n:`@ident`. The
-   premises of this hypothesis (either universal quantifications or
-   non-dependent implications) are instantiated by concrete terms coming either
-   from arguments :n:`{* @term}` or from :ref:`bindings`.
-   In the first form the application to :n:`{* @term}`  can be partial. The
-   first form is equivalent to :n:`assert (@ident := @ident {* @term})`. In the
-   second form, instantiation elements can also be partial. In this case the
-   uninstantiated arguments are inferred by unification if possible or left
-   quantified in the hypothesis otherwise. With the :n:`as` clause, the local
-   hypothesis :n:`@ident` is left unchanged and instead, the modified hypothesis
-   is introduced as specified by the :token:`simple_intropattern`. The name :n:`@ident`
-   can also refer to a global lemma or hypothesis. In this case, for
-   compatibility reasons, the behavior of :tacn:`specialize` is close to that of
-   :tacn:`generalize`: the instantiated statement becomes an additional premise of
-   the goal. The ``as`` clause is especially useful in this case to immediately
-   introduce the instantiated statement as a local hypothesis.
-
-   .. exn:: @ident is used in hypothesis @ident.
+.. tacn:: specialize @one_term_with_bindings {? @as_ipat }
+
+   *First*, the tactic generates a modified term:
```

Comment:
Your splitting of the description with *First* / *Then* makes a lot of sense and is easy to read, but I think the tactic is now missing a general one-sentence description. Something like:

>Specializes a Coq term (typically a hypothesis or a lemma) by applying it to arguments.